tokio/src/signal/unix.rs

```diff
@@ -84,6 +84,17 @@ impl SignalKind {
         Self(signum as libc::c_int)
     }
 
+    /// Get the signal's numeric value.
+    ///
+    /// ```rust,no_run
+    /// # use tokio::signal::unix::SignalKind;
+    /// let kind = SignalKind::interrupt();
+    /// assert_eq!(kind.value(), libc::SIGINT);
+    /// ```
+    pub fn value(&self) -> std::os::raw::c_int {
```

Comment:
I'd recommend calling this `as_raw_value` or something similar (analogous to `as_raw_fd` etc.) just to highlight the distinction more